js/app/modules/contentGroup/mediaLightbox.js

```diff
@@ -57,6 +57,8 @@ const MediaLightbox = new Module.Class({
     },
 
     _on_close: function () {
+        this.lightbox_widget.destroy();
```

Comment:
I guess it will play until it gets GC

You can stop playback with this.lightbox_widget.stop() and let the GC do its work eventually :S